devito/finite_differences/derivative.py

```diff
@@ -222,7 +222,7 @@ def evaluate(self):
 
     def _eval_fd(self, expr):
         expr = getattr(expr, 'evaluate', expr)
-        if self.side in [left, right] and self.deriv_order == 1:
+        if self.side in [left, right, centered] and self.deriv_order == 1:
```

Comment:
nitpicking: neater as ``if self.side is not None`` ?

Reply:
Clearer like that what the options can be IMO